### ISSUE TITLE
[Idea] Moving shows not recorded in awhile to archive section

### ISSUE THREAD

**Issue by Xmetalfanx** [JupiterBroadcasting/plugin.video.jupiterbroadcasting#76]
Just an idea and I know some of the shows are "on hiatus" for now but with some of them having such a big gap since the last episode was recorded and they are not seemly (**) coming back soon ... perhaps see with the folks, at JB think about some shows (I am thinking Tech Talk Today, Faux Show, and Womens Tech Radio) maybe moved to the archived section.    This of course DOESN'T MEAN they are never coming back and if they do come back a few lines can be changed to put them where they were again.

The idea is to ... with the possible exception of LAS depending on how people feel at JB about that ... clean up the addon to show the modern shows that are being recorded weekly.

Just an idea

**Comment by RobLoach** [comment]
Seems like a good idea for Tech Talk Today, Faux Show, and Womens Tech Radio ....

Like you said, may be good to hold off on moving LAS for a bit.

**Comment by rickyphewitt** [comment]
Yea. looks like the jb site has a few archived that this addon still shows as active. I think we should just keep with what is consistent with the JB site.

**Comment by MichaelTunnell** [comment]
I think this is a good idea to archive the shows that are no longer active. I think LAS should also be archived because it's officially over.

> I think we should just keep with what is consistent with the JB site.

Agreed and I've now completed moving LAS to Archive and LAN is now in the show menu of the JB website.

**Comment by Xmetalfanx** [comment]
Really I think staying consistent with the JB site and asking say Chris and/or Rikai what they think would be the way to go.  That is what I'd do before changing anything really

**Comment by MichaelTunnell** [comment]
> Really I think staying consistent with the JB site and asking say Chris and/or Rikai what they think would be the way to go. That is what I'd do before changing anything really

Or in this case, I have edit control over the website so I just moved the stuff to the Archive menu that needed to be.

**Comment by Xmetalfanx** [comment]
About to submit a PR  with a few things moved in the addon.   I have not tested it but I see no reason to with VERY minor edits like the ones I made.   I was looking at the JB site to make sure what I did (as if what shows are active are not common sense) lined up with the site.

Edit: Done ... though I am seeing my branch is 1 ahead (the PR) and 1 Behind this branch .. not sure what happened but if I do any minor changes down the road, I rather make sure everything is even..
